### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ayushgarg22**
i have added my name

### CODE REVIEW

**Comment by Sidddharthh** [comment]
merge approved

File: README.md [hunk 1]
```diff
@@ -13,3 +13,4 @@ Vipul Gupta
 Here are my contributions to this project - ;-)
 @omi10859
 [omkar.site](omkar.site)
+hello this is ayush garg
```
Comment (vipulgupta2048) [inline README.md]: Thanks for the change

Comment (fauzank339) [reply README.md]: Awesome change!

**Comment by creepsshell** [comment]
wow

**Pull request merged**